pkg/state/create.go

```diff
@@ -293,9 +293,15 @@ func (c *StateCreator) scatterGatherEnvSecretFiles(st *HelmState, envSecretFiles
 		},
 		func(id int) {
 			for secret := range secrets {
+				urlOrPath := secret.path
+				localPath, err := c.remote.Locate(urlOrPath)
```

Comment:
if err != nil
then?

Reply:
then urlOrPath := secret.path

the code is copied as-is from `values` process